akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowWireTapSpec.scala

```diff
@@ -19,7 +20,11 @@ class FlowWireTapSpec extends StreamSpec {
   "A wireTap" must {
 
     "call the procedure for each element" in assertAllStagesStopped {
-      Source(1 to 3).wireTap(x ⇒ { testActor ! x }).runWith(Sink.ignore).futureValue
+      Source(1 to 3).throttle(1, 100.millis)
```

Comment:
hm, I see that `WireTap` has one pending slot via `pendingTap`.
When using the full wireTap Sink I guess one can mitigate this by adding a buffer stage after tap.

Wonder how surprising/inconvenient this dropping will be for the function version?
Shall we automatically add a buffer between the tap and the function?

Reply:
@ktoso @johanandren I debugged this to understand what is going on. It's the fuzzing that is causing the dropping, not slowness (blocking) of `f`. Should we simply run this test without fuzzing?

```
class FlowWireTapSpec extends StreamSpec("akka.stream.materializer.debug.fuzzing-mode = off")
```

Reply:
> Wonder how surprising/inconvenient this dropping will be for the function version?
> Shall we automatically add a buffer between the tap and the function?

Yeah was contemplating that but then wasnt sure what to default to so let it be... 



> Should we simply run this test without fuzzing?

Ah great find; yeah let's do that. I was confused why it would act-as-if-blocking and had forgotten about the fuzzer... Will change the test

Reply:
I'd also increase the number of elements in this test to 100